app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -1,38 +1,36 @@
-- if current_user.admin? && @membership_application.status == 'Godkänd'
-  = f.label :membership_number, 'Medlemsnummer', class: 'required'
+- if current_user.admin? && @membership_application.status == t('membership_applications.accepted')
```

Comment:
Since the value of `status` stored in the DB will be in Swedish, won't this logic break for an English locale?